packages/runtime-dom/src/components/Transition.ts

```diff
@@ -200,12 +200,12 @@ export interface ElementWithTransition extends HTMLElement {
 }
 
 export function addTransitionClass(el: ElementWithTransition, cls: string) {
-  el.classList.add(cls)
+  el.classList.add(...cls.split(' '))
```

Comment:
How about `cls.split(' ').forEach(c => el.classList.add(c))` so that we don't need to worry about IE11 compat later?